noxfiles/docker_nox.py

```diff
@@ -185,6 +187,37 @@ def push(session: nox.Session, tag: str) -> None:
         session.run("docker", "tag", sample_app_prod, sample_app_dev, external=True)
         session.run("docker", "push", sample_app_dev, external=True)
 
+    if tag == "git-tag":
+        # if we have an existing git tag on the current commit, we push up
+        # a set of images that's tagged specifically with this git tag.
+        # this publishes images that correspond to commits that have been explicitly tagged,
+        # e.g. RC builds, `beta` tags on `main`, `alpha` tags for feature branch builds.
+        existing_commit_tag = get_current_tag(existing=True)
+        if existing_commit_tag is None:
+            session.log(
+                "Did not find an existing git tag on the current commit, not pushing git-tag images"
+            )
+
+        session.log(
+            f"Found git tag {existing_commit_tag} on the current commit, pushing corresponding git-tag images!"
+        )
+        custom_image_tag = f"{IMAGE}:{existing_commit_tag}"
+        # Push the ethyca/fides image, tagging with :dev
+        session.run("docker", "tag", fides_image_prod, custom_image_tag, external=True)
+        session.run("docker", "push", custom_image_tag, external=True)
+
+        # Push the extra images, tagging with :dev
+        #   - ethyca/fides-privacy-center:dev
+        #   - ethyca/fides-sample-app:dev
+        privacy_center_dev = f"{PRIVACY_CENTER_IMAGE}:{existing_commit_tag}"
+        sample_app_dev = f"{SAMPLE_APP_IMAGE}:{existing_commit_tag}"
+        session.run(
+            "docker", "tag", privacy_center_prod, privacy_center_dev, external=True
+        )
+        session.run("docker", "push", privacy_center_dev, external=True)
+        session.run("docker", "tag", sample_app_prod, sample_app_dev, external=True)
+        session.run("docker", "push", sample_app_dev, external=True)
```

Comment:
nit: your variable names and comments aren't lining up with the actual tags you are using here 😄 
```suggestion
        # Push the ethyca/fides image, tagging with the git tag, e.g. `ethyca/fides:<tag>`
        session.run("docker", "tag", fides_image_prod, custom_image_tag, external=True)
        session.run("docker", "push", custom_image_tag, external=True)

        # Push the extra images, tagging with :<tag>
        #   - ethyca/fides-privacy-center:<tag>
        #   - ethyca/fides-sample-app:<tag>
        privacy_center_custom_image_tag = f"{PRIVACY_CENTER_IMAGE}:{existing_commit_tag}"
        sample_app_custom_image_tag = f"{SAMPLE_APP_IMAGE}:{existing_commit_tag}"
        session.run(
            "docker", "tag", privacy_center_prod, privacy_center_custom_image_tag, external=True
        )
        session.run("docker", "push", privacy_center_custom_image_tag, external=True)
        session.run("docker", "tag", sample_app_prod, sample_app_custom_image_tag, external=True)
        session.run("docker", "push", sample_app_custom_image_tag, external=True)
```

Reply:
😬 copy-pasta'd too fast, thanks for catching